packages/hoc-input/src/formInput.js

```diff
@@ -35,7 +35,8 @@ export const formInputProps = {
   moreInfoContent: PropTypes.node,
   moreInfoAlign: PropTypes.oneOf(values(alignments)),
   prefix: PropTypes.string,
-  suffix: PropTypes.string
+  suffix: PropTypes.string,
+  autoFocus: PropTypes.bool
```

Comment:
The name of the prop is dubious. It sounds like it would focus the input on mount.

How about `autoFocusHighlight` or `autoFocusStyled`? I'm not 100% about these, but I couldn't think of anything better.

Reply:
Maybe `autoControlFocusedStyle`?

Reply:
OK, sound good!